lib-rpc/ocamlformat_rpc_lib.mli

```diff
@@ -9,70 +9,78 @@
 (*                                                                        *)
 (**************************************************************************)
 
-module type Command_S = sig
-  type t
+module Make (IO : IO.S) : sig
+  module type Command_S = sig
+    type t
 
-  val read_input : Stdlib.in_channel -> t
+    val read_input : IO.ic -> t IO.t
 
-  val to_sexp : t -> Sexplib0.Sexp.t
+    val to_sexp : t -> Csexp.t
 
-  val output : Stdlib.out_channel -> t -> unit
-end
+    val output : IO.oc -> t -> unit IO.t
+  end
 
-module type Client_S = sig
-  type t
+  module type V = sig
+    module Command : Command_S
 
-  type cmd
+    module Client : sig
+      type t
```

Comment:
All but `type t` can be removed from `Client` ?

Except if `pick_client` isn't suitable for every users, but in this case I would suggest having an other constructor function for the other use cases.

Reply:
If we scrap these functions from the `Client` module it doesn't compile anymore